src/cli/mod.rs

```diff
@@ -1,17 +1,17 @@
 use crate::cli::pcap::PcapDumpOpts;
 use crate::cli::unit::UnitOpts;
+use crate::cli::wallet::WalletOpts;
 use crate::debug::parse_pcap_log_file_to_csv;
 use crate::node::node_with_single_peer;
 use address::AddressOpts;
-use anyhow::{anyhow, Context};
+use anyhow::anyhow;
```

Comment:
I remember @gak saying we should not use anyhow. I'm still learning here, what's the main advantage of anyhow?

Reply:
Let's discuss in discord :)